.github/workflows/qa.yaml

```diff
@@ -18,35 +18,40 @@ on:
 env:
   DEBIAN_FRONTEND: noninteractive
   GO_TESTS_TIMEOUT: 20m
-  apt_deps: >-
+  c_build_dependencies: >-
+    clang-tools
+    clang
+    libglib2.0-dev
     libpam-dev
+
+  go_build_dependencies: >-
     libglib2.0-dev
+    libpam-dev
     libpwquality-dev
 
-  test_apt_deps: >-
+  go_test_dependencies: >-
     apparmor-profiles
     bubblewrap
     cracklib-runtime
     git-delta
     openssh-client
-    openssh-server
-
-  # In Rust the grpc stubs are generated at build time
-  # so we always need to install the protobuf compilers
-  # when building the NSS crate.
-  protobuf_compilers: >-
-    protobuf-compiler
+    openssh-server  
 
 jobs:
   go-sanity:
     name: "Go: Code sanity"
     runs-on: ubuntu-24.04 # ubuntu-latest-runner
     steps:
+      - uses: actions/checkout@v5
+      - name: Disable installation of locales and manpages
+        run: .github/scripts/dpkg-exclude-locales-and-manpages.sh
       - name: Install dependencies
         run: |
+          # Install dependencies
+          set -eu
+
           sudo apt update
-          sudo apt install -y ${{ env.apt_deps }}
-      - uses: actions/checkout@v5
+          sudo apt install -y ${{ env.go_build_dependencies }}
```

Comment:
Small nitpick since we are already touching the actions: we should probably switch to using `apt-get` rather than `apt` for the CI runs, wdyt?

Reply:
yes, I was considering doing that in all our actions. I'll prepare a PR.